src/js/src/components/ToolboxComponents/DomainStats.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <div class="domainStatsContainer">
     <h2 class="toolboxHeadline">
-      Domain stats
+      {{ $t('toolBox:domain_stats') }}
     </h2>
     <div class="domainContentContainer">
       <input v-model="domain"
-             placeholder="Enter domain, like 'kb.dk'"
+             :placeholder="$t('toolBox:enter_domain')"
              :class="$_checkDomain(domain) ? '' : 'urlNotTrue'"
              @keyup.enter="loadGraphData(domain)">
       <button :disabled="loading" class="domainStatsButton" @click.prevent="loadGraphData(domain)">
-        Generate
+        {{ $t('toolBox:generate') }}
```

Comment:
See my comment above regarding leveraging object hierarchy instead of the key:key notation